### PR TITLE
Cfitsio 3.470 rebuild

### PR DESCRIPTION
**Pull request by tobijk**
Rebuild against libgfortran5

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -41,22 +42,24 @@ test:
...
-    - gdal  # [x86_64 and linux]
+  # Disable downstream "gdal" until gdal itself has been rebuilt.
+  # downstreams:
+      # - gdal  # [x86_64 and linux]
 
 about:
-  home: http://heasarc.gsfc.nasa.gov/fitsio/fitsio.html
-  license: fitsio
+  home: https://heasarc.gsfc.nasa.gov/fitsio/fitsio.html
+  license: ISC
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: I see that the ISC license is a little bit different from the license used by cfitsio:
- ISC https://spdx.org/licenses/ISC.html
- cfitsio https://heasarc.gsfc.nasa.gov/FTP/software/fitsio/c/License.txt

Probably that's why conda-forge uses `LicenseRef-fitsio`: https://github.com/conda-forge/cfitsio-feedstock/blob/fd4a4354a41191e926b85d0e93ae068661bac0df/recipe/meta.yaml#L52

**Review by skupr-anaconda — COMMENTED** [review]
LGTM

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (tobijk) [reply recipe/meta.yaml]: You're right, I'll get a second opinion.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -17,8 +17,9 @@ source:
     - lcurl-pkg-config.patch
 
 build:
-  number: 6
```
Comment (katietz) [inline recipe/meta.yaml:20]: Shouldn't we bump build-number here?

**Review by katietz — COMMENTED** [review]
Otherwise looks good

File: recipe/meta.yaml (hunk 2 above)
Comment (tobijk) [reply recipe/meta.yaml:20]: Uhm, it's bumped to 7 two lines further below?

**Review by katietz — COMMENTED** [review]
lgtm

**Pull request merged**